### PR TITLE
Testing a branch

### PR DESCRIPTION
**Pull request by jamt23**
## Overview

This is my completed challenge for Day 01

I did not know how to make it cleaner so I made a different parameter for each object and named them all block with a number and then just created a div for each one. 

The hint you provided kind of tricked me but then I figured it out that it was the border radius amount for each one that you were providing.

### CODE REVIEW

File: challenges/day-01/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            border-radius: 35%;
+        } .block4 {
+            height:100px;
+            width:100px;
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            border-radius: 50%;
+        }
```
Comment (nilsonmolina) [inline challenges/day-01/index.html]: This works great, but notice how there is a lot of duplicated code? We have a principle in programming known as the DRY principle, which stands for **D**on't **R**epeat **Y**ourself.

If I had changed my mind and said that I want the blocks to each be 200px, you'd have to change it for every single block here. So here's a proposed alternative:

```suggestion
        .block {
            height:100px;
            width:100px;
            background-color:#fa7e70;
            margin: 10px 0 10px 0;
        }
        .block.smooth {
            border-radius: 10px;
        }
        .block.rounded {
            border-radius: 35%;
        }
        .block.circle {
            border-radius: 50%;
        }
```

**Comment by nilsonmolina** [comment]
Also, I noticed there was a `.DS_Store` file in this push. Those are files that macOS automatically creates, but we don't want those in our repos. Preventing certain files from being saved to our repos is a very important topic. The short of it is that we can use a `.gitignore` file to define what files should and should not be included in the repo.

To make this easy, I'll walk you through a basic .gitignore file.

1. Create a file named `.gitignore` in the projects root directory (the web-basics folder).
2. Add the following lines of text to the .gitignore file:
    ```
    .*
    !/.gitignore
    ```

This essentially tells git to ignore any files that begin with a period, but allow any files named `.gitignore`.

File: challenges/day-01/index.html (hunk 1 above)
Comment (jamt23) [reply challenges/day-01/index.html]: Thank you so much for that advice. I applied the DRY protocol as per your suggestion and it looks alot cleaner and works better.

**Review by nilsonmolina — APPROVED** [review]
Looks great!

**Pull request merged**